CoolCurrencyTests/CurrencyViewModelTests.swift

```diff
@@ -94,6 +94,15 @@ class CurrencyViewModelTests: XCTestCase {
     }
     
     class MockDatabaseRepository: DatabaseRepositable {
+        
+        func updateUsersScoreboard(SignedInUser number: Int, name userName: String, finalScore userFinalScore: String, totalScore userTotalScore: String, completion: @escaping DatabaseResponse) {
+            
+        }
+        
+        func retrieveUserScoreboards(completion: @escaping (Result<[LeadershipBoardDataModel], Error>) -> Void) {
```

Comment:
Little bit neater to have these functions like this:
`func retrieveUserScoreboards(completion: @escaping (Result<[LeadershipBoardDataModel], Error>) -> Void) { }`

Just optional though.